spot-client/src/spot-tv/ui/views/meeting.js

```diff
@@ -82,9 +83,9 @@ export class Meeting extends React.Component {
         this._useJwt = false;
 
         if (this._queryParams.location && props.jwt) {
-            const { host } = new URL(this._queryParams.location);
-
-            this._useJwt = props.jwtDomains.includes(host);
+            this._useJwt = this._useJwt = Boolean(findWhitelistedMeetingUrl(
```

Comment:
what is the meaning of `this._useJwt = this._useJwt` ?

Reply:
bad copy/paste